### PR TITLE
During experiment time tool page

### PR DESCRIPTION
**Pull request by gadorlhiac**
Early draft outline, not yet ready.

### CODE REVIEW

**Review by fredericpoitevin — COMMENTED** [review]
Thanks Gabriel, this is a fantastic improvement from existing documentation! I wish I had something like this last year :D 

I do have some comments that we can discuss. Whenever I started it with "TODO" should be made an issue and addressed in a later PR. The rest I'd like to address now.

File: docs/during/timetool.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
+# Timing Ultrafast Pulses
+If you are performing a pump-probe measurement requiring sub-picosecond time resolution, you will need to correct for the jitter in the X-ray arrival time. LCLS deploys a time tool to facilitate this. The theory of the tool is detailed [here](/before/timetool). Below you will find a step-by-step guide for setting up the tool, and ensuring it is calibrated and running well for your experiment.
```
[the remark this answers is not in the document]
Comment (fredericpoitevin) [reply docs/during/timetool.md:2]: Created issue #5

File: docs/during/timetool.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+Before the time tool can be employed it must be calibrated. During the process of cross-correlation a "time zero" was determined. This involved setting the position of an optical delay stage. That position is now known as the temporal zero in the DAQ, and distances can be set relative to it. By moving the delay stage so that the optical pulse arrives at discrete times before, or after, the X-ray pulse we can build up a calibration curve. Due to the jitter, for every nominal target time there will be a distribution of actual relative arrival times; however, by targetting a set of nominal delays we can fit a calibration curve through the centroids of these distributions. Refer to the theory [here](/before/timetool) if the concept and terminology are unfamiliar to you.
+
+**Procedure**:
+
+1. Determine the number corresponding to the time tool calibration run. Either the beam line scientist or person(s) working the DAQ controls can tell you if you do not know.
+2. Edit the btx YAML file to include parameters for a new task ```calibrate_timetool```. Refer to the [btx introduction](/before/btxintro) if you are unclear on which file this is. This task takes three parameters:
```
[the remark this answers is not in the document]
Comment (fredericpoitevin) [reply docs/during/timetool.md:21]: Created issue #6

File: docs/during/timetool.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+  model:
+  figs: True
+```
+<ol start=3>
+<li>Submit your task using <code>elog_submit.sh</code>.</li></ol>
+```bash
+$ elog_submit.sh -c ../yamls/myyaml.yaml -t timetool_correct
+```
+<ol start=4>
+<li>Repeatedly submit the task with different sets of runs as needed. Monitor diagnostic figures to ensure that timing is not drifting.</li></ol>
```
[the remark this answers is not in the document]
Comment (fredericpoitevin) [reply docs/during/timetool.md:84]: Create issue #7

File: docs/during/timetool.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (gadorlhiac) [reply docs/during/timetool.md:2]: Switching to `Setup` and `Analysis`

File: docs/during/timetool.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
+# Timing Ultrafast Pulses
+If you are performing a pump-probe measurement requiring sub-picosecond time resolution, you will need to correct for the jitter in the X-ray arrival time. LCLS deploys a time tool to facilitate this. The theory of the tool is detailed [here](/before/timetool). Below you will find a step-by-step guide for setting up the tool, and ensuring it is calibrated and running well for your experiment.
+
+## Controls
```
[the remark this answers is not in the document]
Comment (gadorlhiac) [reply docs/during/timetool.md:4]: Added a slightly different version that includes some of those ideas and is more like a TL;DR
` This page, similarly to others, is divided into two sections, **Setup** and **Analysis**. The **Setup** section of the page contains information on the procedure to perform optical pulse/X-ray pulse cross-correlation for establishing "time-zero" - when both pulses arrive at the sample simultaneously - as well as collecting the dataset used for time tool calibration. The **Analysis** section details how to analyze the calibration dataset to establish a jitter correction model, as well as how to apply that model to create accurate time stamps for events corresponding to experimental data.`

File: docs/during/timetool.md [hunk 5]
```diff
@@ -0,0 +1,94 @@
+# Timing Ultrafast Pulses
```
[the remark this answers is not in the document]
Comment (gadorlhiac) [reply docs/during/timetool.md:1]: We should discuss names further.

Switching this title to "Running the timetool". "Timing Ultrafast Pulses" sounds like a good page title for the before>timetool page.

File: docs/during/timetool.md [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+If you are performing a pump-probe measurement requiring sub-picosecond time resolution, you will need to correct for the jitter in the X-ray arrival time. LCLS deploys a time tool to facilitate this. The theory of the tool is detailed [here](/before/timetool). Below you will find a step-by-step guide for setting up the tool, and ensuring it is calibrated and running well for your experiment.
+
+## Controls
+### Cross-correlation
```
[the remark this answers is not in the document]
Comment (gadorlhiac) [reply docs/during/timetool.md:5]: Switched to `Setup`

File: docs/during/timetool.md [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+Before running the time tool it needs to be established that "time zero" at the sample interaction point is the same as the time tool's. The beams on both the timing and pump arms are synchronized; however, the variable path that they travel needs to be accounted for. This is accomplished through the use of delay stages. A cross-correlation measurement usually in cerium-yttrium aluminium garnet (Ce-YAG), will be used to confirm that the optical pulses arrive with the same x-ray pulse at both the sample IP and the time tool. The beam-line scientists and your point of contact will be able to confirm what material is available for you. The following section outlines the necessary procedure for performing the cross-correlation.
+
+(TBA with Controls)
+
+### Calibration Runs
+
+## Software
```
[the remark this answers is not in the document]
Comment (gadorlhiac) [reply docs/during/timetool.md:12]: Switch to `Analysis` . Include TLDR in the intro paragraph to explain the sections more clearly

File: docs/during/timetool.md [hunk 8]
```diff
@@ -0,0 +1,94 @@
...
+
+!!! danger "Timing Drifts"
+    The instrument timing will drift over the course of your experiment. Make sure to frequently calibrate the timetool and only use recent calibration models for performing timetool corrections.
+
+**Procedure**:
+
+1. Determine the run, or runs, for which you would like to generate timetool corrections. Any number of contiguous runs can be analyzed in a single task submission, as long as all the runs in the specified range were collected with the **same nominal time delay**.
+2. Create a new task, ```timetool_correct```, in the btx YAML file. This task takes four parameters:
+    - ```runs``` : A single run or range of runs to correct with the time tool. Ranges should be specified with a hyphen. E.g. 17-21
+    - ```nominal``` : The nominal delay in picoseconds.
```
[the remark this answers is not in the document]
Comment (gadorlhiac) [reply docs/during/timetool.md:64]: Fine by me.

I'll update here as well as the corresponding code in btx

File: docs/during/timetool.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (fredericpoitevin) [reply docs/during/timetool.md]: Thanks!

File: docs/during/timetool.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (fredericpoitevin) [reply docs/during/timetool.md]: Perfect, thanks!

File: docs/during/timetool.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (fredericpoitevin) [reply docs/during/timetool.md]: OK

File: docs/during/timetool.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (fredericpoitevin) [reply docs/during/timetool.md]: Thanks!

**Review by fredericpoitevin — APPROVED** [review]
Thanks Gabriel, it's a good start for this page.

**Pull request merged**